perf-tests/src/coreapi.py

```diff
@@ -31,13 +41,39 @@ def contains_alternate_node(json_resp):
         return bool(result) and isinstance(result, list) \
             and (result[0].get('recommendation', {}) or {}).get('alternate', None) is not None
 
-    def start_stack_analysis(self):
-        """Start the stack analysis, sending the manifest file."""
-        filename = 'data/requirements_click_6_star.txt'
+    @staticmethod
+    def get_manifest_name(filename):
+        """Get the standard manifest name for the given filename."""
+        manifests = {
+            ".txt": "requirements.txt",
+            ".xml": "pom.xml",
+            ".json": "package.json"}
+        for extension, manifest in manifests.items():
+            if filename.endswith(extension):
+                return manifest
+        raise "Unknown extension in filename: {f}".format(f=filename)
+
+    @staticmethod
+    def prepare_manifest_files(filename):
+        """Send the selected manifest file to stack analysis."""
+        # default variable substitutions
+        filename = filename or 'requirements_click_6_star.txt'
+        manifest_name = CoreApi.get_manifest_name(filename)
+
+        print("filename with manifest data: {m}".format(m=manifest_name))
+        print("standard manifest name: {n}".format(n=manifest_name))
+
+        filename = 'data/{filename}'.format(filename=filename)
```

Comment:
this means that manifest files have to be stored always inside `data` folder. Is it your intention?

Reply:
yes, to make perf.tests easier to run